src/UniversalDashboard/Server/DashboardHub.cs

```diff
@@ -215,7 +215,8 @@ public async Task UnregisterEvent(string eventId)
             {
                 variables.Add("EventData", eventData);
             }
-
+            variables.Add("EventId", EventId);
```

Comment:
@oitptobbe It looks like the build failed because EventId is capital but the parameter passed into the hub is lower case.

Reply:
Of course :)

Its been done.